doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst

```diff
@@ -0,0 +1,1256 @@
+  * Replaced the :kconfig:option:`CONFIG_BT_L2CAP_TX_BUF_COUNT` Kconfig option with :kconfig:option:`CONFIG_BT_ATT_TX_COUNT` in nRF Desktop dongle configurations.
+    This update is needed to align with the new approach introduced in ``sdk-zephyr`` by commit ``a05a47573a11ba8a78dadc5d3229659f24ddd32f``.
+  * The :ref:`nrf_desktop_hid_forward` no longer uses a separate HID report queue for a HID peripheral connected over Bluetooth LE.
+    The module relies only on the HID report queue of a HID subscriber.
+    This is done to simplify implementation, reduce memory consumption and speed up retrieving enqueued HID reports.
+    You can modify the enqueued HID report limit through the :ref:`CONFIG_DESKTOP_HID_FORWARD_MAX_ENQUEUED_REPORTS <config_desktop_app_options>` Kconfig option.
+  * ``Error while sending report`` log level in :ref:`nrf_desktop_hid_state` from error to warning.
+    The log might appear, for example, during the disconnection of a HID transport.
+    In that case, it does not denote an actual error.
+  * Updated the number of ATT buffers (:kconfig:option:`CONFIG_BT_ATT_TX_COUNT`) for nRF Desktop peripherals.
+    This adjustment allows peripherals to simultaneously send all supported HID notifications (including HID report pipeline support), the BAS notification, and an ATT response.
+    ATT uses a dedicated net buffer pool.
+
+Matter Bridge
+-------------
+
+* Added:
+
+  * The :ref:`CONFIG_BRIDGE_BT_MAX_SCANNED_DEVICES <CONFIG_BRIDGE_BT_MAX_SCANNED_DEVICES>` Kconfig option to set the maximum number of scanned Bluetooth LE devices.
+  * The :ref:`CONFIG_BRIDGE_BT_SCAN_TIMEOUT_MS <CONFIG_BRIDGE_BT_SCAN_TIMEOUT_MS>` Kconfig option to set the scan timeout.
+
+* Updated the implementation of the persistent storage to leverage ``NonSecure``-prefixed methods from the common Persistent Storage module.
+* Changed data structure of information stored in the persistent storage to use fewer settings keys.
+  The new structure uses approximately 40% of the memory used by the old structure, and provides a new field to store user-specific data.
+  Backward compatibility is kept by using an internal dedicated method that automatically detects the older data format and performs data migration to the new representation.
+
+IPC radio firmware
+------------------
+
+* Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <samples>`.
+
+* Added new categories of samples: :ref:`dect_samples` and :ref:`suit_samples`.
+
+Bluetooth samples
+-----------------
+
+* Added:
+
+  * The :ref:`bluetooth_iso_combined_bis_cis` sample showcasing forwarding isochronous data from CIS to BIS.
+  * The :ref:`bluetooth_isochronous_time_synchronization` sample showcasing time-synchronized processing of isochronous data.
+  * Support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board in the :ref:`fast_pair_input_device` sample.
```

Comment:
this entry should be removed as it is moved to the new category "Bluetooth Fast Pair samples" and we have a duplication here